views/partials/stock/product/directives/create-asset-form.jade

```diff
@@ -6,14 +6,14 @@ div(ng-controller="AssetController as assetVM")
           .form-group(ng-init="assetVM.setForm(assetForm)")
             .row
               .col-sm-8
-                label #{ __("Digital Asset Type") }
+                label #{ __("Data asset type") }
                 select.form-control(name="asset", ng-model="assetVM.digitalChars[0].productSpecCharacteristicValue[0].value", ng-change="assetVM.setCurrentType()")
                   option(ng-repeat="asset in assetVM.assetTypes", value="{{ asset.name }}") {{ asset.name }}
-              .col-sm-4
-                label #{ __("How to provide?") }
-                select.form-control(ng-model="assetVM.currFormat", ng-options="format for format in assetVM.currentType.formats")
+              //.col-sm-4
```

Comment:
Can you remove the code rather than commenting? We can always find back the code in git, if ever needed